db/seeds.rb

```diff
@@ -5,3 +5,64 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+User.create([ 
+  {username:"dj",
+   password_digest: "dj",
+   role: 1,
+   full_name: "Dj",
+   email: "djloveshisgf@gmail.com"},
+
+   {username: "fred",
```

Comment:
Align the elements of an array literal if they span more than one line.<br>Space inside { missing.